src/frontend/src/hooks/UseImportSession.tsx

```diff
@@ -9,6 +9,7 @@ import { useInstance } from './UseInstance';
  * Custom hook for managing the state of a data import session
  */
 
+// TODO: Load these values from the server?
 export enum ImportSessionStatus {
```

Comment:
@SchrodingersGat could this just be a generic state?

Reply:
Defined via the API, you mean, like a "stock status" or similar?

Reply:
Yes, that would make it pretty flexible